Source/WebKit/Shared/ios/NativeWebTouchEventIOS.mm

```diff
@@ -113,26 +107,25 @@ static CGFloat radiusForTouchPoint(const _UIWebTouchPoint& touchPoint)
         auto radius = radiusForTouchPoint(touchPoint);
         platformTouchPoint.setRadiusX(radius);
         platformTouchPoint.setRadiusY(radius);
-        platformTouchPoint.setRotationAngle(0); // Not available in _UIWebTouchEvent yet.
+        platformTouchPoint.setRotationAngle(0); // Not available in WKTouchPoint yet.
```

Comment:
Can we add this? If so, can we make a radar and a FIXME: instead?

Reply:
I filed [a new bug](https://webkit.org/b/259068) to implement this, which I'll reference in this comment with a FIXME (I avoided adding it here, since this patch should just preserve existing behavior).